google-cloud-notification/README.md

```diff
@@ -0,0 +1,77 @@
+Google Cloud Java Client for Firestore
```

Comment:
Update the firestore references in this readme

Reply:
For sure!